blueocean-dashboard/src/main/js/components/LinkifiedText.jsx

```diff
@@ -18,12 +18,14 @@ const LinkifiedText = (props) => {
             const partialLinksRegExpString = new RegExp(`(${partialLinksIdString.slice(0, -1)})`, 'gi');
 
             for (let commitMsgPart of text.split(partialLinksRegExpString)) {
+                commitMsgPart = commitMsgPart.replace(/ /g, '\u00A0');
```

Comment:
Could you please add a comment as to what this unicode character is? Or just assign it to a variable so it self-documents.